proxy/src/http/org.rs

```diff
@@ -0,0 +1,258 @@
+        .and(document::document(document::tag("Org")))
+        .and(document::document(
+            document::response(
+                200,
+                document::body(registry::Org::document()).mime("application/json"),
+            )
+            .description("Successful retrieval"),
+        ))
+        .and(document::document(
+            document::response(
+                404,
+                document::body(super::error::Error::document()).mime("application/json"),
+            )
+            .description("Org not found"),
+        ))
+        .and_then(handler::get)
+}
+
+/// Org handlers for conversion between core domain and http request fullfilment.
+mod handler {
+    use radicle_registry_client::Balance;
+    use std::sync::Arc;
+    use tokio::sync::RwLock;
+    use warp::http::StatusCode;
+    use warp::{reply, Rejection, Reply};
+
+    use crate::notification;
+    use crate::registry;
+
+    /// Register an org on the Registry.
+    pub async fn register(
+        registry: Arc<RwLock<registry::Registry>>,
+        subscriptions: notification::Subscriptions,
+        input: super::RegisterInput,
+    ) -> Result<impl Reply, Rejection> {
+        // TODO(xla): Get keypair from persistent storage.
+        let fake_pair = radicle_registry_client::ed25519::Pair::from_legacy_string("//Alice", None);
+        // TODO(xla): Use real fee defined by the user.
+        let fake_fee: Balance = 100;
+
+        let mut reg = registry.write().await;
+        let tx = reg.register_org(&fake_pair, input.id, fake_fee).await?;
+
+        subscriptions
+            .broadcast(notification::Notification::Transaction(tx.clone()))
+            .await;
+
+        Ok(reply::with_status(reply::json(&tx), StatusCode::CREATED))
+    }
+
+    /// Get the Org for the given `id`.
+    pub async fn get(
+        id: String,
+        registry: Arc<RwLock<registry::Registry>>,
+    ) -> Result<impl Reply, Rejection> {
+        let reg = registry.read().await;
+        let org = reg.get_org(id).await?;
+
+        Ok(reply::json(&org))
+    }
+}
+
+impl Serialize for registry::Org {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut state = serializer.serialize_struct("Org", 2)?;
+        state.serialize_field("id", &self.id)?;
+        state.serialize_field("avatarFallback", &self.avatar_fallback)?;
+
+        state.end()
+    }
+}
+
+impl ToDocumentedType for registry::Org {
+    fn document() -> document::DocumentedType {
+        let mut properties = std::collections::HashMap::with_capacity(2);
+        properties.insert("avatarFallback".into(), avatar::Avatar::document());
+        properties.insert(
+            "id".into(),
+            document::string()
+                .description("The id of the org")
+                .example("monadic"),
+        );
+
+        document::DocumentedType::from(properties).description("Unique org")
```

Comment:
```suggestion
        document::DocumentedType::from(properties).description("Org")
```

Used the singular type everywhere else for now.